drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/renderers/BaseExpressionGridTheme.java

```diff
@@ -67,7 +67,7 @@
 
     @Override
     public String getName() {
-        return "Scenario Simulation Editor";
+        return "Test Scenarios (Preview)";
```

Comment:
Since this is a client side name is it visible to user? If so, then it would also need to be i18nized.

Reply:
I think this value will be visible only if we want to support multiple themes with the possibility for the user to switch. For now I think it is enough to change it here without i18n

Reply:
@Rikkola, thank you for noticing, I don't think this is visible at the moment (as @danielezonca wrote). From the code:

```
/**
 * Definition of themes used by a render for the pluggable rendering mechanism.
 */
public interface GridRendererTheme {

    /**
     * Returns a display name for the theme.
     * @return A name for the theme
     */
    String getName();
...
```